game_catalog/models.py

```diff
@@ -22,9 +32,27 @@ def __str__(self):
 class Game(models.Model):
     name = models.CharField(max_length=100)
     description = models.TextField()
-    release_year = models.IntegerField()
+    release_year = models.IntegerField(validators=[validate_year])
     genre = models.ManyToManyField(Genre)
     studio = models.ForeignKey(Studio, on_delete=models.CASCADE)
 
     def __str__(self):
         return self.name
+
+
+class CustomUser(AbstractUser):
+    favorite_games = models.ManyToManyField(
+        Game, blank=True, related_name="favorited_by"
```

Comment:
More options: `users_who_favorited`, `favorited_by_users`, `users_who_added_to_favorites`
I think that `users_who_added_to_favorites` is clear enough, but it might feel slightly verbose.

Reply:
Initially, I felt that favorited_by sounded a bit... stitchy? But after reviewing the other suggestions, I think it aligns best with the original flow. Since this relationship is inherently tied to users, keeping it concise and descriptive makes the most sense.

Compared to alternatives like users_who_added_to_favorites, favorited_by feels cleaner and more consistent with typical naming conventions. It avoids verbosity while still making the relationship clear. So, I’d say sticking with favorited or the original favorited_by is the best choice here!